src/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+}
+
+.snake {
+  display: grid;
```

Comment:
make it with flex, this solution save for later, we have task anti snake, which should be maded with grid